core/src/main/kotlin/org/evomaster/core/output/naming/RestActionTestCaseNamingStrategy.kt

```diff
@@ -0,0 +1,32 @@
+package org.evomaster.core.output.naming
+
+import org.evomaster.core.problem.httpws.HttpWsCallResult
+import org.evomaster.core.problem.rest.RestCallAction
+import org.evomaster.core.search.EvaluatedIndividual
+import org.evomaster.core.search.Solution
+import org.evomaster.core.search.action.EvaluatedAction
+
+open class RestActionTestCaseNamingStrategy(
+    solution: Solution<*>,
+    languageConventionFormatter: LanguageConventionFormatter
+) : ActionTestCaseNamingStrategy(solution, languageConventionFormatter)  {
+
+    override fun expandName(individual: EvaluatedIndividual<*>, nameTokens: MutableList<String>): String {
+        var evaluatedAction = individual.evaluatedMainActions().last()
+        var action = evaluatedAction.action as RestCallAction
```

Comment:
see previous comments: why `var`?

Reply:
Yep, these are clearly mistakes. Got to get used to the Kotlin var/val approach, my bad

Reply:
Fixed